viewer/static/css/experiment-metadata.styl

```diff
@@ -0,0 +1,33 @@
+@import 'lib';
+
+$grid-padding = $min-padding;
+
+#experiment-metadata {
```

Comment:
Eh, I'll have to agree to disagree. I think the specificity rules work well when used appropriately. But I don't mind changing it.